Calculator/CalculatorLibrary/CalculatorLibrary.cs

```diff
@@ -0,0 +1,78 @@
+﻿using System;
+using System.IO;
+using System.Diagnostics;
+using Newtonsoft.Json;
+
+namespace CalculatorLibrary
+{
+    public class Calculator
+    {
+        JsonWriter writer;
+
+        // Constructor incl opening Json Writer & file for log of operations
+        public Calculator()
+        {
+            StreamWriter logFile = File.CreateText("calculatorlog.json");
```

Comment:
Better to use

```cs
using(StreamWriter logfile = File.CreateText("calculatorlog.json")
{
//CODE TO OPERATE THE STREAMWRITER HERE
}
```

This ensures you dispose of the writer, otherwise you run the risk of a memory leak, even with the GC